models/Notification/Service/NotificationServiceFilterParser.php

```diff
@@ -110,11 +110,11 @@ private function parseString(array $item): array
     {
         $result = null;
         $property = $this->getDbProperty($item);
-        $value = $item[self::KEY_VALUE];
+        $value = array_key_exists(self::KEY_VALUE, $item) ? $item[self::KEY_VALUE] : "";
```

Comment:
We don't have to consider `null` here, right?
```suggestion
        $value = $item[self::KEY_VALUE] ?? '';
```